docs/es/docs/tutorial/body.md

```diff
@@ -0,0 +1,164 @@
+
+<img src="/img/tutorial/body/image03.png">
+
+Además, tendrás señalamiento de errores para las operaciones que sean incorrectas:
+
+<img src="/img/tutorial/body/image04.png">
+
+No se trata de una casualidad, **FastAPI** ha sido construido entorno a este diseño.
+
+Y ha sido probado a fondo durante la etapa de diseño, antes de realizar cualquier implementación, con el fin de asegurar que funcionará en todos los editores.
+
+Para conseguir esto, incluso ha sido necesario realizar cambios en Pydantic.
+
+Las capturas de pantalla anteriormente mostradas han sido obtenidas de<a href="https://code.visualstudio.com" class="external-link" target="_blank">Visual Studio Code</a>.
+
+Sin embargo, todas las funcionalidades son completamente compatibles con <a href="https://www.jetbrains.com/pycharm/" class="external-link" target="_blank">PyCharm</a> y la mayoría de editores para Python:
+
+<img src="/img/tutorial/body/image05.png">
+
+!!! tip
+Si estás utilizando <a href="https://www.jetbrains.com/pycharm/" class="external-link" target="_blank">PyCharm</a> como editor, puedes usar el<a href="https://github.com/koxudaxi/pydantic-pycharm-plugin/" class="external-link" target="_blank">Plugin de Pydantic para PyCharm</a>.
+
+    El uso de los modelos Pydantic mejoran la ayuda del editor en:
+    * autocompletado
+    * verificación de tipos
+    * refactorización
+    * búsquedas
+    * inspecciones
+
+## Uso del modelo
+
+Tendrás acceso a todos los atributos del objecto del modelo desde dentro de la función:
+
+```Python hl_lines="21"
+{!../../../docs_src/body/tutorial002.py!}
+```
+
+## Cuerpo de la Petición (Request body) + parámetros de ruta (Path)
+
+Puedes declarar al mismo tiempo parámetros de ruta y peticiones con datos en el cuerpo (**request body**)
+
+**FastAPI** reconocerá tanto los parámetros de la función que deban ser **obtenidos desde la ruta (Path)**, así como aquellos parámetros que deban formar parte del modelo de Pydantic y que tengan que **obtenerse del cuerpo de la petición (Request Body)**
+
+```Python hl_lines="17-18"
+{!../../../docs_src/body/tutorial003.py!}
+```
+
+## Cuerpo de la Petición (Request body) + ruta (Path) + parámetros de consulta (Query)
+
+También puedes declarar al mismo tiempo, parámetros de cuerpo (**body**), ruta (**path**) y consulta (**query**)
+
+**FastAPI** reconocerá cada uno de ellos y recogerá los datos del sitio correcto.
+
+```Python hl_lines="18"
+{!../../../docs_src/body/tutorial004.py!}
+```
+
+Los parámetros de la función serán reconocidas de la siguente forma:
+
+- Si el parámetro está declarado en la ruta (**path**), será utilizado como parámetro de ruta.
+- Si el parámetro es un **tipo singular**, como entero (`int`), decimal (`float`), cadena de caracteres (`str`), booleano (`bool`), etcétera) serán interpretados como consulta (**query**)
```

Comment:
There is a mismatched bracket after `etcétera`, also do you think `float` should be referred as `flotante` instead of `decimal` so it does not get confused with the [decimal](https://docs.python.org/3/library/decimal.html) standard library?